diff --git a/VisionCamera.h b/VisionCamera.h
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/VisionCamera.podspec b/VisionCamera.podspec
index 739478fc1573be37faf48410b9dd9c7568fea7bf..0dd4951d499b66deb05579fee619e5315f6bcea1 100644
--- a/VisionCamera.podspec
+++ b/VisionCamera.podspec
@@ -43,6 +43,7 @@ Pod::Spec.new do |s|
   # All source files that should be publicly visible
   # Note how this does not include headers, since those can nameclash.
   s.source_files = [
+    "VisionCamera.h",
     "ios/**/*.{m,mm,swift}",
     "ios/CameraBridge.h",
     "ios/Skia Render Layer/PreviewSkiaView.h",
